wizard/src/utils/obs_generation.ts

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import _ from "lodash";
+
+import {
+  Column,
+  CsvData,
+  MappedThing,
+  Mapping,
+  MappingType,
+  MappingVal,
+  Observation,
+  RowObservations,
+} from "../types";
+
+// Helper maps used while generating observaitons.
```

Comment:
nit: typo at the end of this comment. s/observaitons/observations

Reply:
fixed.